code/modules/mob/stats_skills.dm

```diff
@@ -188,16 +187,13 @@ proc/conToToxinModifier(var/constitution, var/w_class)
 
 /mob/proc/learn_skills(var/skill_type)
 	var/initial_skill = round(skills[skill_type])
-	if(stat_to_modifier(stats["int"]) > 0) // This is still based off int
-		skills[skill_type] += 0.01 * stat_to_modifier(stats["int"])
-	else 
-		if(skills[skill_type] < 30)
+	if(skills[skill_type] < 30 && stat_to_modifier(stats["int"]) >= 10) //the minimum for reading
```

Comment:
stat_to_modifier(stats["int"]) >= 10 should be  stat_to_modifier(stats["int"]) >= 0, because we are checking the modifier not the stat it's self